front/public/locales/en/rollingstock.json

```diff
@@ -22,10 +22,11 @@
   "electricOnly": "Electric only",
   "equipement": "Equipment",
   "errorMessages": {
-    "missingInformation": "Missing information",
-    "must be number": "Number is required",
-    "must be equal to one of the allowed values": "Authorised value required",
-    "rollingStockUsed": "Rolling stock is used in the following scenarios:",
+    "missingInformation": "missing informations",
+    "must be number": "must be a number",
+    "minMaxRangeError": "the value must be between {{min}} and {{max}}",
```

Comment:
Why did you remove all the capital letters ?

Reply:
I forgot to set capital letters, done